common/app/views/fragments/analytics/base.scala.html

```diff
@@ -28,4 +28,15 @@
     <img src="@Configuration.debug.beaconUrl/count/pv.gif" alt="" style="display : none ;" rel="nofollow"/>
     <img src="https://www.facebook.com/tr?id=@FBPixel.account&ev=PageView&noscript=1" height="1" width="1" style="display:none" rel="nofollow" alt="" />
 
+    <!-- Twitter universal website tag code -->
```

Comment:
Do we want to add the script tag to the base analytics? Could it load as a commercial third-party-tag?

The [third-party-tags module](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/commercial/modules/third-party-tags.js) also has a helper function for loading scripts, and each tag has the flow-type:

```js
type ThirdPartyTag = {
    shouldRun: boolean,
    url: string,
    onLoad?: () => any,
    useImage?: boolean,
};
```

With this loading pattern is that we often base the `shouldRun` around a switch:

```js
export const inizio: ThirdPartyTag = {
    shouldRun: config.get('switches.inizio', false),
    url:
        '//cdn.brandmetrics.com/survey/script/e96d04c832084488a841a06b49b8fb2d.js',
    onLoad,
};
```

This makes it a bit faster and easier to enable/disable a particular tag.